### ISSUE TITLE
Monthly financial statement detail Ecommerce

### ISSUE THREAD

**Issue by afs1jes** [afs1jes/Clarins-CA-IR-Migration#17]

**Comment by afs1jes** [comment]
-- experimenting with selectable inside a @ member cell

**Comment by afs1jes** [comment]
corrected display column headers for lag formulas.
Removed zoom Control (improved builds from 30 seconds to 10 seconds).
added Do Not Read/Write markers to spacer columns.

**Comment by FrederickGiroux** [comment]
Formating not the same, missing the grouping for the ''+'' or ''-'' to collapse and expand

**Comment by afs1jes** [comment]
edited grid properties to allow outlining. Revoked subtotal formulas for all columns. Locked all cells in grid (grid is not write-enabled). 

Returning for retest.

**Comment by FrederickGiroux** [comment]
Column spacing too wide

**Comment by A3Huntley** [comment]
Migrated formatting again to correct small discrepancies.

**Comment by FrederickGiroux** [comment]
Workbook name is different. Otherwise its ok

**Comment by FrederickGiroux** [comment]
![Image](https://user-images.githubusercontent.com/124090756/234072926-35295f83-8610-4309-abfc-bec4161f6554.png)

**Comment by A3Huntley** [comment]
Yeah, how strange. Seems like maybe the CONCAT formula I'd used for the WorkbookName name was messing with that. I changed the formula to use ampersands instead (like most other templates), and that seems to do the trick.



![Image](https://user-images.githubusercontent.com/88676502/234082466-d4373cd0-340f-4176-a111-7c875b97a54b.png)
